libc/math/static-vector-data.h

```diff
@@ -0,0 +1,1540 @@
+/*
+ * Phoenix-RTOS
+ *
+ * POSIX.1-2017 standard library functions tests
+ *
+ * Static vector tests data
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Adam Debek
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <float.h>
+
+/*
+ * DO NOT INCLUDE THIS FILE ANYWHERE ELSE EXCEPT STATIC-VECTOR.C !!!!!!!
+ * DO NOT INCLUDE THIS FILE ANYWHERE ELSE EXCEPT STATIC-VECTOR.C !!!!!!!
+ * DO NOT INCLUDE THIS FILE ANYWHERE ELSE EXCEPT STATIC-VECTOR.C !!!!!!!
+ * DO NOT INCLUDE THIS FILE ANYWHERE ELSE EXCEPT STATIC-VECTOR.C !!!!!!!
+ * DO NOT INCLUDE THIS FILE ANYWHERE ELSE EXCEPT STATIC-VECTOR.C !!!!!!!
+ */
```

Comment:
No point in screaming if noone is listening. How about:
```
#ifndef TEST_USE_STATIC_VERTOR_H_
#error Do not include static-vector-data.h anywhere except static-vector.c
#endif
```
+ `define TEST_USE_STATIC_VERTOR_H_` in static-vector.c

Reply:
Or potential size getters for vectors and declare it as extern